pkgs/misc/emulators/desmume/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchurl
+, pkgconfig, libtool, intltool
+, libXmu
+, lua
+, agg, alsaLib, soundtouch, openal
+, desktop_file_utils
+, gtk2, gtkglext, libglade, pangox_compat
+, mesa, mesa_glu, libpcap, SDL, zziplib }:
+
+with stdenv.lib;
```

Comment:
It is my particular idiom :)

In general, I noted when I build my expressions I always needed "stdenv.lib" in the most outside scope. So, that's it.